package.json

```diff
@@ -9,16 +9,20 @@
   },
   "dependencies": {
     "apollo-cache-inmemory": "^1.6.2",
+    "babel-polyfill": "^6.26.0",
```

Comment:
You shouldn't have to modify the package file unless you're updating dependencies. These newly added dependencies don't seem to fit in here.